reana/reana_dev/git.py

```diff
@@ -330,8 +331,13 @@ def git_fork(component, exclude_components, browser):  # noqa: D301
 @click.option(
     "--exclude-components", default="", help="Which components to exclude? [c1,c2,c3]",
 )
+@click.option(
+    "--target-directory",
+    callback=validate_directory,
+    help="In which directory install? [.]",
```

Comment:
* We don't need the `[.]` as there's no default value anymore.
* What about a slightly different wording? Something like "In which directory to clone?" In the end, we're not installing anything but just cloning repositories